pynest/examples/pynest-microcircuit/network_params.py

```diff
@@ -0,0 +1,192 @@
+    # Number of neurons in the different populations. The order of the
+    # elements corresponds to the names of the variable 'populations'.
+    'N_full': np.array([20683, 5834, 21915, 5479, 4850, 1065, 14395, 2948]),
+    # Mean rates of the different populations in the non-scaled version
+    # of the microcircuit. Necessary for the scaling of the network.
+    # The order corresponds to the order in 'populations'.
+    'full_mean_rates':
+        np.array([0.971, 2.868, 4.746, 5.396, 8.142, 9.078, 0.991, 7.523]),
+    # Connection probabilities. The first index corresponds to the targets
+    # and the second to the sources.
+    'conn_probs':
+        np.array(
+            [[0.1009, 0.1689, 0.0437, 0.0818, 0.0323, 0., 0.0076, 0.],
+             [0.1346, 0.1371, 0.0316, 0.0515, 0.0755, 0., 0.0042, 0.],
+             [0.0077, 0.0059, 0.0497, 0.135, 0.0067, 0.0003, 0.0453, 0.],
+             [0.0691, 0.0029, 0.0794, 0.1597, 0.0033, 0., 0.1057, 0.],
+             [0.1004, 0.0622, 0.0505, 0.0057, 0.0831, 0.3726, 0.0204, 0.],
+             [0.0548, 0.0269, 0.0257, 0.0022, 0.06, 0.3158, 0.0086, 0.],
+             [0.0156, 0.0066, 0.0211, 0.0166, 0.0572, 0.0197, 0.0396, 0.2252],
+             [0.0364, 0.001, 0.0034, 0.0005, 0.0277, 0.008, 0.0658, 0.1443]]),
+    # Number of external connections to the different populations.
+    # The order corresponds to the order in 'populations'.
+    'K_ext': np.array([1600, 1500, 2100, 1900, 2000, 1900, 2900, 2100]),
+    # Factor to scale the indegrees.
+    'K_scaling': 0.1,
+    # Factor to scale the number of neurons.
+    'N_scaling': 0.1,
+    # Amplitude of excitatory postsynaptic potential.
+    'PSP_e': 0.15,
+    # Standard deviation of the postsynaptic potential.
+    'PSP_sd': 0.1,
+    # PSP mean matrix
+    'PSP_mean_matrix': get_mean_PSP_matrix(0.15, -4, 8),
+    # PSP std matrix
+    'PSP_std_matrix': get_std_PSP_matrix(0.1, 8),
+    # mean delay matrix.
+    'mean_delay_matrix': get_mean_delays(1.5, 0.75, 8),
+    # std delay matrix.
+    'std_delay_matrix': get_std_delays(0.75, 0.375, 8),
+    # Rate of the poissonian spike generator.
+    'bg_rate': 8.,
+    # Turn Poisson input on or off (True or False).
+    'poisson_input': True,
+    # Delay of the poisson generator
+    'poisson_delay': 1.5,
+    # Parameter of the neurons.
+    'n_params': {
+        # Membrane potential average for the neurons (in mV).
+        'V0_mean': -58.0,
+        # Standard deviation of the average membrane potential (in mV).
+        'V0_sd': 10.0,
+        # Resting potential of the neurons (in mV).
+        'E_L': -65.0,
+        # Threshold potential of the neurons (in mV).
+        'V_th': -50.0,
+        # Membrane potential after a spike (in mV).
+        'V_reset': -65.0,
+        # Membrane capacitance (in pF).
+        'C_m': 250.0,
+        # Membrane time constant (in ms).
+        'tau_m': 10.0,
+        # Time constant of postsynaptic excitatory currents (in ms).
+        'tau_ex': 0.5,
```

Comment:
Here, 'tau_syn_ex' and 'tau_syn_in' would be better to keep these parameter names consistent with the NEST names. If you keep these parameter names strictly consistent, then you can just pass this dictionary to `nest.SetStatus` in the `Network` class, where you set the neuron parameters.
For this, you have to define `V0_mean` and `V0_std` outside of this dictionary.